adapters/syslog/syslog.go

```diff
@@ -67,7 +77,10 @@ func NewSyslogAdapter(route *router.Route) (router.LogAdapter, error) {
 	default:
 		return nil, errors.New("unsupported syslog format: " + format)
 	}
-	tmpl, err := template.New("syslog").Parse(tmplStr)
+
+	funcMap := template.FuncMap{
+		"getEnvVar":   tplGetEnvVar}
```

Comment:
I don't really understand your comment here. What I'm trying to do is create a new template function so you can get environment variables from the slice .Container.Config.Env , as you can see in the first code bit in my first comment.